server/plugin.go

```diff
@@ -224,9 +253,30 @@ func (p *Plugin) createSNSRdsEventAttachment(subject string, messageNotification
 	return attachment
 }
 
+func (p *Plugin) createSNSCloudformationEventAttachment(subject string, messageNotification SNSCloudformationEventNotification) model.SlackAttachment {
+	p.API.LogDebug("AWSSNS HandleNotification Cloudformation Event", "MESSAGE", subject)
```

Comment:
Seems `SUBJECT` would be a better choice here

```suggestion
	p.API.LogDebug("AWSSNS HandleNotification Cloudformation Event", "SUBJECT", subject)
```